internal/compiler/passes/const_propagation.rs

```diff
@@ -98,6 +98,29 @@ fn simplify_expression(expr: &mut Expression) -> bool {
                 }
                 ('|', Expression::BoolLiteral(false), e) => Some(std::mem::take(e)),
                 ('|', e, Expression::BoolLiteral(false)) => Some(std::mem::take(e)),
+                ('>', Expression::NumberLiteral(a, un1), Expression::NumberLiteral(b, un2))
```

Comment:
Could also work for ≤ and ≥

Reply:
I was wondering how >= fits in one character, but I guess I got my answer lol